### PR TITLE
Just some output for help fix

### PR DESCRIPTION
**Pull request by DantestyleXD**
Added some new output

I comment them for let Parser work normally

### CODE REVIEW

**Comment by collett8192** [comment]
Actually the idea is to proxy every event call into onProgress and do work there, and arguments should be called param1 param2 etc. instead of actor actorId because it is reused for other things between different event call. I'll change them tomorrow. And when fixing quests containing mob killing, comment out [this line](https://github.com/collett8192/Sapphire/blob/the_movie/FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs#L55) so it will assign progress to actual bnpcs instead of moving them to eobjects.

File: FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void GenerateQuestScript()
             outputCpp.Add("{");
             outputCpp.Add("public:");
             outputCpp.Add($"  {questId}() : Sapphire::ScriptAPI::EventScript( {questNumber} ){{}}; ");
-            outputCpp.Add($"  ~{questId}() = default; ");
+            outputCpp.Add($"  ~{questId}() {{}} ");
```
Comment (collett8192) [inline FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs:88]: What does this do? I think the default destructor is fine.

Comment (DantestyleXD) [reply FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs:88]: It's a habit, in truth it doesn't change anything but I can't see it with default 🙈

Comment (DantestyleXD) [reply FFXIVTheMovie/FFXIVTheMovie/ParserV3/Parser.cs:88]: In anycase, if this PR doesn't need you can close this

**Pull request closed without merging**